testdata/intake-v3/rum_events.ndjson

```diff
@@ -1,6 +1,3 @@
+{"me": {"y": {"t": "Processing"},"sa": {"ysc": {"v": 1},"yss": {"v": 124}}}}
```

Comment:
Is that a valid case that a metricset belongs to a span but not to a transaction?

Reply:
Yeah, probably doesn't make much sense, but I wanted to keep a top level event to test they are ingested. RUM doesn't send system metrics yet (they will soon) so there aren't  "real" examples we can use.